typings/index.d.ts

```diff
@@ -1600,6 +1601,11 @@ declare module 'discord.js' {
 
 	type AwaitReactionsOptions = ReactionCollectorOptions & { errors?: string[] };
 
+	type BanInfo = {
+		user: User;
+		reason?: string;
```

Comment:
`reason` is always present, but may be null
```suggestion
		reason: string | null;
```